docs/section6/Rule6-7.md

```diff
@@ -0,0 +1,53 @@
+
+# Lighting - Rule 6-7
+
+**Rule ID:** 6-7  
+**Rule Description:** Where a complete lighting system exists and where a lighting system has been designed and submitted with design documents, the baseline LPD is equal to expected value in Table G3.7. Where lighting neither exists nor is submitted with design documents, baseline LPD shall be determined in accordance with Table G3-8.
+
+**Appendix G Section:** Section G3.1-6 Modeling Requirements for the Baseline
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+**Manual Check:** Yes
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Table G3.7 and Table G3.8  
+**Function Call:**  
+
+  - get_lighting_status_type()
+  - match_data_element()
+  - data_lookup()
+
+
+## Rule Logic: 
+
+- For each building segment in the baseline model: `building_segment_b in B_RMR.building.building_segments:`  
+
+  - Get matching building segment in R_RMR: `building_segment_p = match_data_element(P_RMR, BuildingSegments, building_segment_b.id)`
+
+    - Get lighting status type dictionary for P_RMR: `space_lighting_status_type_dict_p = get_lighting_status_type(building_segment_p)`  
+  
+  - For each thermal block in building segment: `thermal_block_b in building_segment_b.thermal_blocks:`  
+  
+    - For each zone in thermal block: `zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `space_b in zone_b.spaces:`  
+
+        - For each space in zone: `space_b in zone_b.spaces:`  
+
+          - Get total lighting power density in space: `total_space_LPD_b = sum(interior_lighting.power_per_area for interior_lighting in space_b.interior_lighting)`
+
+          - Get lighting status type for space: `space_lighting_status_type = space_lighting_status_type_dict_p[match_data_element(P_RMR, Spaces, space_b.id).id]`
+
+            **Rule Assertion:**
+
+            - Case 1: If space lighting status type is as-designed or as-existing, and space total interior lighting power density in B_RMR matches Table G3.7: `if ( space_lighting_status_type == "AS-DESIGNED OR AS-EXISTING" ) AND ( total_space_LPD_b == data_lookup(table_G3_7, space_b.lighting_space_type) ): PASS`  
+
```

Comment:
The baseline must be based on Table G3.7 irrespective of the proposed lighting status type, so I think we don't need to check for "space_lighting_status_type == "AS-DESIGNED OR AS-EXISTING"" in Case 1, just that the baseline total space LPD is based on Table G3.7. So case 1 and case 3 should be merged.

Reply:
ok, merging case 1 and case 3 makes sense.

Reply:
@mkarpman Case 3 is for when lighting is design/existing but the LPD also matches Table 9.5.1 value, as the function right now cannot differentiate this case and not-yet designed. Let me know how we want to treat this case. Similar question for Rule 6-4.

Reply:
The rule description in row #5 suggests that "Where lighting neither exists nor is submitted with design documents, baseline LPD shall be determined in accordance with Table G3-8.", but I don't see this language in 90.1 2019. This was introduced in a later addendum. So for 2019 version, the baseline is always based on Table G3-7.

Reply:
@mkarpman i don't quite understand this requirement in 2019 App G. If space types are not known, how can the baseline LPD be set based on G3.7? there would be no lighting space type specified in such scenarios, correct?